src/main/java/frc/robot/subsystems/elevator/io/ElevatorIO.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.subsystems.elevator.io;
+
+import java.util.function.DoubleSupplier;
+
+import frc.lib.logfields.IOBase;
+import frc.lib.logfields.LogFieldsTable;
+
+public abstract class ElevatorIO extends IOBase{
+public final DoubleSupplier encoderValue = fields.addDouble("encoderValue", this::getEncoder);
```

Comment:
height. not encoder value. the io shouldnt be specific for the hardware. also value is almost never needed in a name, all variables contain value.